package.json

```diff
@@ -110,6 +110,7 @@
     "nsp": "^2.2.0",
     "pre-commit": "^1.1.2",
     "spectron": "^0.36.0",
+    "sqlite3": "^3.1.1",
```

Comment:
we don't need this in browser packaging right? Just asking so I can exclude from the installer packaging.